docs/feature_flags.md

```diff
@@ -125,7 +125,40 @@ histogram (albeit via the text format). With this flag enabled, Prometheus will
 still ingest those conventional histograms that do not come with a
 corresponding native histogram. However, if a native histogram is present,
 Prometheus will ignore the corresponding conventional histogram, with the
-notable exception of exemplars, which are always ingested.
+notable exception of exemplars, which are always ingested. To keep the
+conventional histograms as well, enable `scrape_classic_histograms` in the
+scrape job.
+
+NOTE: Due to the protobuf serialization, conventional histogram `le` labels
+and summary `quantile` labels will be formatted differently. Any integer
+label values will be represented as a float, thus `1` becomes `1.0`, `2` becomes
+`2.0` and so on. This changes the identity of the resulting series in storage.
+This affects for example applications instrumented with client_golang provided
+that [promhttp.HandlerOpts.EnableOpenMetrics](https://pkg.go.dev/github.com/prometheus/client_golang/prometheus/promhttp#HandlerOpts)
+is set to false.
+
+As a side effect of conventional histogram labels change is that aggregations
+such as `rate`, `irate`, `increase` or `histogram_quantile` spanning the time
+range when the labels changed will show the wrong results. On the other hand
+going forward metrics will have the same labels regardless of instrumentation
+library.
```

Comment:
There are more problems (aggregation, dashboards, alerts). I'm not sure if we should enumerate the problems. Just say that this creates all kind of problems during the transition. I would mention, though, that the same problem happens if OpenMetrics is activated in client_golang, and then finish with the sentence already there (that the formatting is now unified, and also include here that it has converged onto OpenMetrics format to avoid further inconsistencies).

Reply:
I've reworded the paragraph, ptal.